packages/client/src/components/MovieCard/MovieCard.jsx

```diff
@@ -42,6 +65,28 @@ export function MovieCard({ movie, favorites, toggleFavorite }) {
         </StyledTypographyReviewText>
       </StyledLink>
       <BasicRating rating={movie.rating} />
+      <Snackbar
+        open={snackbarOpen}
+        autoHideDuration={1500}
+        onClose={handleSnackbarClose}
+        message={
+          snackbarMessage === 'added' ? (
```

Comment:
If you are doing this sort of comparison on text, it's worth to think about maybe creating an enum or a object with key and text, so that you make sure that even if your text changes, the logic will remain untacked (aka separating content from logic).

I would suggest something easy like this:
```
const messages = {
positive: "Here comes positive",
negative: "Here comes negative"
}
```
and then you can use them (obviously) by keys:
```
if (message === message.positive)
```

Of course this is pseudocode so name it as you please :)
```

Reply:
It even looks more readable, thank you for the tip :)